packages/lisk-transactions/src/5_create_dapp.js

```diff
@@ -38,7 +38,7 @@ const validateOptions = options => {
 };
```

Comment:
We should probably also have minimal checks for the other properties.

Reply:
@willclarktech Are they optional keys and we already have schema check. Is it really necessary ?

Reply:
If the schema check was sufficient we wouldn't need this issue at all. A thorough solution will include these keys. Feel free to create a follow-up issue if you think we should merge this first and address them separately.